modules/installation-aws-route53.adoc

```diff
@@ -44,5 +44,4 @@ uses. For example, if you registered your domain to a Route53 service in a
 different accounts, see the following topic in the AWS documentation:
 link:https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-name-servers-glue-records.html#domain-name-servers-glue-records-procedure[Adding or Changing Name Servers or Glue Records].
 
-. If you use a subdomain, follow your company's procedures to add its delegation
-records to the parent domain.
+. If you are using a subdomain, add its delegation records to the parent domain. This gives Amazon Route53 responsibility for the subdomain. Follow the delegation procedure outlined by the parent domain's DNS provider. See link:https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingNewSubdomain.html[Creating a subdomain that uses Amazon Route 53 as the DNS service without migrating the parent domain] in the AWS documentation for an example high level procedure.
```

Comment:
s/parent domain's DNS provider/DNS provider of the parent domain/

Per IBM SG about avoiding `’s` for possessive form of inanimate objects

Reply:
Thank you Alex! Resolved.